### PR TITLE
feat: Move tips and information to ticket info

### PR DESCRIPTION
**Pull request by KristianSelnas**
Closes https://github.com/AtB-AS/kundevendt/issues/15900

Some of the files that appear to be new in this PR are actually just moved with minor changes. It's just that Git got at bit confused along the way.

### CODE REVIEW

File: src/tips-and-information/TipsAndInformation.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      {tips.map((tip, index) => {
+        const title = getTextForLanguage(tip.title, language);
+        const emoji = tip.emoji;
+        const description = getTextForLanguage(tip.description, language);
+
+        if (!emoji || !title || !description) return null;
+
+        return (
+          <ExpandableSectionItem
+            key={index}
```
Comment (marius-at-atb) [inline src/tips-and-information/TipsAndInformation.tsx:48]: Should the key instead be `title + description`?

File: src/translations/screens/subscreens/PurchaseOverview.ts [hunk 2]
```diff
@@ -264,6 +264,11 @@ const PurchaseOverviewTexts = {
         'Ticket description',
         'Billettbeskriving',
       ),
+      tipsInformation: _(
+        'Generelt om billetter',
+        'About tickets in general',
+        'Generelt om bilettar',
```
Comment (marius-at-atb) [inline src/translations/screens/subscreens/PurchaseOverview.ts:270]:
```suggestion
        'Generelt om billettar',
```

**Review by marius-at-atb — APPROVED** [review]
Lgtm!

File: src/tips-and-information/TipsAndInformation.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            const newTips = mapToTips(snapshot.docs);
+            const sortedTips = sortTipsByTitle(newTips, language);
+            setTips(sortedTips);
+          },
+          (err) => {
+            console.warn(err);
+          },
+        ),
+    /*
+    Resubscribing on language change? Doh. Note that we shouldn't have
```
Comment (hanne-at-atb) [inline src/tips-and-information/TipsAndInformation.tsx:31]: Is this comment meant to be here?

**Review by hanne-at-atb — APPROVED** [review]
Cool 😄

File: src/tips-and-information/TipsAndInformation.tsx (hunk 3 above)
Comment (KristianSelnas) [reply src/tips-and-information/TipsAndInformation.tsx:31]: This is not code that's written for this pr. It's a file that's moved, but Git couldn't keep up through various rebases etc. The original code was written by the students.

File: src/tips-and-information/TipsAndInformation.tsx (hunk 1 above)
Comment (KristianSelnas) [reply src/tips-and-information/TipsAndInformation.tsx:48]: Most likely. However, as commented below, this code was not added now - it is just moved. This piece was written by the students.

File: src/tips-and-information/TipsAndInformation.tsx (hunk 3 above)
Comment (hanne-at-atb) [reply src/tips-and-information/TipsAndInformation.tsx:31]: Ahh, okey

Comment (KristianSelnas) [reply src/tips-and-information/TipsAndInformation.tsx]: Made a fix anyways: https://github.com/AtB-AS/mittatb-app/pull/4142/commits/94e7f9e8891a7239ce3260f191d93802c5afd5e7

File: src/tips-and-information/TipsAndInformation.tsx (hunk 1 above)
Comment (KristianSelnas) [reply src/tips-and-information/TipsAndInformation.tsx]: Fixed

**Pull request merged**